src/client/java/minicraft/level/tile/StairsTile.java

```diff
@@ -40,7 +40,7 @@ public boolean mayPass(Level level, int x, int y, Entity e) {
 	public boolean interact(Level level, int xt, int yt, Player player, Item item, Direction attackDir) {
 		super.interact(level, xt, yt, player, item, attackDir);
 
-		// Makes it so you can remove the stairs if you are in creative and debug mode.
+		// Makes it so you can remove the stairs if you are in creative mode.
 		if (item instanceof PowerGloveItem && Game.isMode("minicraft.settings.mode.creative")) {
```

Comment:
Are you able to place stairs in creative?

Reply:
We could add the ability then.

Reply:
Did you or did you not remove the debug option?

Reply:
There was a debug check for this, but I think it should be able also without debug.

Reply:
You shouldn't be able to destroy stairs in non-debug mode as you can only place them with the debug menu. Don't add a new item!

Reply:
Ok.